### ISSUE TITLE
Griefers getting away with reporting comm abuse for no reason

### ISSUE THREAD

**Issue by eric1294** [ValveSoftware/Dota2-Gameplay#10290]
### Description

I just lost 1.5k behavior score despite not being terribly angry, now it's down to 1.5k or something. Limit the amount of reports that someone with lower than 3000 behavior score can send out!

**Comment by Pallu08** [comment]
Well well, if it isn't the consequences of your own actions.

**Comment by ValveHatesDotA** [comment]
Behaviour score should be removed since it serves no purpose other than to let griefers report others and screw them over even more by having them be matched more often with griefers. Actual griefing should result in bans. Abusive chat should either not be punished (you can just block them) or just result in a mute. Valve is griefing ppl

**Comment by BenLubar** [comment]
> Behaviour score should be removed since it serves no purpose other than to let griefers report others and screw them over even more by having them be matched more often with griefers. Actual griefing should result in bans. Abusive chat should either not be punished (you can just block them) or just result in a mute. Valve is griefing ppl

If your behavior score is low, that means the system thinks you are likely to grief, cheat, or be abusive. All behavior score does is match people who are likely to be toxic together and people who are unlikely to be toxic together. It does not change the gameplay, and it does not affect any rewards.

Either you believe that low behavior score is where people who are likely to be toxic end up, which means the system is working, or you believe that behavior score has no correlation to toxicity, which doesn't seem to be what you are claiming.

If behavior score was removed as a matchmaking criteria, more matches would have at least one game-ruining toxic player.

**Comment by ValveHatesDotA** [comment]
>All behavior score does is match people who are likely to be toxic together and people who are unlikely to be toxic together. It does not change the gameplay, and it does not affect any rewards.

No. Behaviour score => Griefers are matched with real players causing the real players to be griefed. The real players can't easily increase their behaviour score either because the griefers will report them because they're griefers and reporting people for nothing is perfectly valid because the report system doesn't work.

**Comment by OwLester** [comment]
BRO i already like you. You are the normal one just keep it up FUCK ALL OF THEM just don't communicate with retards play your own game and leave a likes who play good (even with a low KDA) and report every dumb schoolar who thinks he s a "PRO". BE STRONG, keep your vision clear and be rational one (embrace stoicism)
